Doc/library/ast.rst

```diff
@@ -329,6 +329,13 @@ and classes for traversing abstract syntax trees:
    numbers and column offsets are not dumped by default.  If this is wanted,
    *include_attributes* can be set to true.
 
+   If *indent* is a non-negative integer or string, then the tree will be
+   pretty-printed with that indent level.
```

Comment:
I don't think the docs here make it clear what the string version of the argument does. Can you add in a little line to explain it just like in `JSONEncoder`'s documentation?